examples/chaplin-brunch/.gitignore

```diff
@@ -1,8 +1,28 @@
-bower_components/
-!bower_components/todomvc-common/
+# NPM packages.
+node_modules/chaplin/*
+!node_modules/chaplin/chaplin.js
+
+node_modules/exoskeleton/*
+!node_modules/exoskeleton/exoskeleton.js
+
+node_modules/backbone.localstorage/*
+!node_modules/backbone.localstorage/backbone.localStorage.js
+
+node_modules/backbone.nativeview/*
+!node_modules/backbone.nativeview/backbone.nativeview.js
+
+node_modules/todomvc-app-css/*
+!node_modules/todomvc-app-css/index.css
+
+node_modules/todomvc-common/*
+!node_modules/todomvc-common/base.css
+!node_modules/todomvc-common/base.js
+
+node_modules/coffee-script-brunch/
+node_modules/javascript-brunch/
+node_modules/handlebars-brunch/
+node_modules/uglify-js-brunch/
```

Comment:
If you want to ignore a whole folder, you should [have an asterisk](https://github.com/tastejs/todomvc/pull/1142).